### DISCUSSION TITLE
setDataRaw - Used to stuff SBC bytes into stream?

### DISCUSSION THREAD

**Discussion by SamObde** [pschatzmann/ESP32-A2DP#636] [Q&A] [answered]
Hi!

Great library!  I am working on a project where I have stored SBC audio data - not PCM data.  Can I replay the SBC bytes on a source without having to decode to PCM first?  I see there is a setDataRaw in SoundData class but I am not sure if thats what its for or how to use it.

Thanks!

**Comment by pschatzmann** [comment, marked as answer]
No, only PCM data is supported, so you need to decode it first.
